src/resources/views/crud/form_content.blade.php

```diff
@@ -12,6 +12,9 @@
   </div>
 @endif
 
+@php
```

Comment:
This allow us to mark parent crud already loaded fields when loading the related create modal.